bin/release

```diff
@@ -19,6 +19,8 @@ rake release
 
 docker build --rm -t codeclimate/codeclimate .
 docker push codeclimate/codeclimate
+docker tag codeclimate/codeclimate "codeclimate/codeclimate:$version"
+docker push "codeclimate/codeclimate:$version"
```

Comment:
Nice :+1: 

I think that  `docker push` will push all tagged images so you can move the `docker tag` line up and drop the second `docker push`.

```bash
 docker build --rm -t codeclimate/codeclimate .
 docker tag codeclimate/codeclimate codeclimate/codeclimate:$version
 docker push codeclimate/codeclimate
```